docs/.gitkeep

```diff
@@ -0,0 +1 @@
+
```

Comment:
So I gather that this is traditionally a dummy file to make sure Git keeps a directory, but since there are other files in `docs/` is it actually needed? Or maybe there's new meaning to it that my quick Google search didn't find :-)

Reply:
oh actually yes, that can be removed... I believe I added that when first creating the `docs` folder :sweat_smile: